### DISCUSSION TITLE
胡桃！！！

### DISCUSSION THREAD

**Discussion by L-walnut** [share121/inter-knot#3318] [其他]
![屏幕截图 2024-08-20 195332](https://github.com/user-attachments/assets/8889d1ea-8703-486e-a31d-52c290822891)
![屏幕截图 2024-08-20 195358](https://github.com/user-attachments/assets/486901b6-4cfa-492d-b666-a37d40ec2810)
![屏幕截图 2024-08-20 195449](https://github.com/user-attachments/assets/52c9c079-1aaa-4efc-bf46-157a9d4f0561)
![屏幕截图 2024-08-20 195523](https://github.com/user-attachments/assets/90ea2e22-0372-4769-ad3f-db1f746f68ed)
![屏幕截图 2024-08-20 195550](https://github.com/user-attachments/assets/843a7a4b-4033-448b-9e07-c1d9cdbce396)
![屏幕截图 2024-08-20 195632](https://github.com/user-attachments/assets/0acc3bbb-7617-4b1a-9542-27adeb27f161)
![屏幕截图 2024-08-20 195956](https://github.com/user-attachments/assets/b70efa55-a35c-4796-8626-0966044b802a)
![屏幕截图 2024-08-20 200054](https://github.com/user-attachments/assets/040ee1d1-39f5-479d-91b8-3045aec2c285)
![屏幕截图 2024-08-20 200123](https://github.com/user-attachments/assets/a8557fd3-7102-4ed3-9351-b2aee3bd877d)
![屏幕截图 2024-08-20 200144](https://github.com/user-attachments/assets/b97e2b2c-0e3c-4e86-a8f1-db6040cd1f7a)
![屏幕截图 2024-08-20 200210](https://github.com/user-attachments/assets/26675fff-9708-4610-9397-1f1644280003)
 图片从网上找取，部分有出处,侵删

**Comment by Nie7bai** [comment]
第二碑半价！<img src="https://bbs-static.miyoushe.com/static/2024/02/06/a06b39a87b3fec5be0f4ecbf4f758cad_5751963257293893910.png" alt="胡桃-收好啦" width="75" >

**Comment by opqphp** [comment]
![20240625_034059](https://github.com/user-attachments/assets/1a117ec8-ee91-4a14-b329-a5ae81139d0d)

**Comment by opqphp** [comment]
磕胡芙吗
![illust_111446009_20240819_004149](https://github.com/user-attachments/assets/1ab82bd8-c4c2-4f34-9e6e-d561be713c67)

**Reply by zzztjs** [reply]
胡胡和芙桃（

**Comment by opqphp** [comment]
![1723981657949](https://github.com/user-attachments/assets/45b7788a-0306-40c4-bd56-32f5f4a59dc3)
bilibili：衫青瑜

**Comment by opqphp** [comment]
啊？是你！这绳网好小啊

**Reply by L-walnut** [reply]
小胡桃大梦想
